### PR TITLE
Add missing translation strings

### PR DESCRIPTION
**Pull request by timarney**
# Summary | Résumé

Adds missing translation strings and removes welcome.json (page was removed)

Strings found using https://marketplace.visualstudio.com/items?itemName=lokalise.i18n-ally

Note: for some reason the "settings" and "preview" strings we're not getting picked up properly via lokalise so I updated the key names.

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [x] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [ ] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [ ] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [ ] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [ ] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [ ] Have you modified the change log and updated any relevant documentation?
- [ ] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [ ] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

File: public/static/locales/en/common.json [hunk 1]
```diff
@@ -98,8 +98,10 @@
     "logout": "Sign out"
   },
   "alert": {
-    "dismissAlert": "Close this alert"
+    "dismissAlert": "Close this alert",
+    "dismissNotification": "Close this alert"
```
Comment (craigzour) [inline public/static/locales/en/common.json:102]: Is it normal that both `dismissAlert` and `dismissNotification` have the same value? If yes then we could just use one of them.

File: public/static/locales/en/common.json [hunk 2]
```diff
@@ -98,8 +98,10 @@
...
   },
   "continue": "Continue",
-  "error": "Error"
+  "error": "Error",
+  "unknown": "unknown"
```
Comment (craigzour) [inline public/static/locales/en/common.json:106]: Is capital "U" needed here?

File: public/static/locales/fr/common.json [hunk 3]
```diff
@@ -99,8 +99,10 @@
     "logout": "Se déconnecter"
   },
   "alert": {
-    "dismissAlert": "Fermer cette alerte"
+    "dismissAlert": "Fermer cette alerte",
+    "dismissNotification": "Close this alert"
```
Comment (craigzour) [inline public/static/locales/fr/common.json:103]: Missing [FR] flag

**Review by craigzour — CHANGES_REQUESTED** [review]
Just found a few little things. Let me know if this is relevant or not :)

File: public/static/locales/fr/common.json [hunk 4]
```diff
@@ -99,8 +99,10 @@
...
   },
   "continue": "Continue [FR]",
-  "error": "Error [FR]"
+  "error": "Error [FR]",
+  "unknown": "unknown"
```
Comment (craigzour) [inline public/static/locales/fr/common.json:107]: Missing [FR] flag

File: public/static/locales/en/common.json (hunk 1 above)
Comment (timarney) [reply public/static/locales/en/common.json:102]: Just going to use dismissAlert --- it's for an aria label should be fine

File: public/static/locales/en/common.json (hunk 2 above)
Comment (timarney) [reply public/static/locales/en/common.json]: Updated to use a capital

https://github.com/cds-snc/platform-forms-client/pull/1467/commits/27e935fbb7d7990ba7353a7b599eb975206bcf67

File: public/static/locales/fr/common.json (hunk 3 above)
Comment (timarney) [reply public/static/locales/fr/common.json]: Fixed:
https://github.com/cds-snc/platform-forms-client/pull/1467/commits/27e935fbb7d7990ba7353a7b599eb975206bcf67

File: public/static/locales/fr/common.json (hunk 4 above)
Comment (timarney) [reply public/static/locales/fr/common.json]: Fixed:
https://github.com/cds-snc/platform-forms-client/pull/1467/commits/27e935fbb7d7990ba7353a7b599eb975206bcf67

**Review by craigzour — APPROVED** [review]
LGTM :)

**Pull request merged**